src/poprox_recommender/model/nrms/__init__.py

```diff
@@ -14,11 +14,15 @@ class NRMS(torch.nn.Module):
     Input 1 + K candidate news and a list of user clicked news, produce the click probability.
     """
 
-    def __init__(self, args):
+    def __init__(self, config):
         super(NRMS, self).__init__()
 
-        self.news_encoder = NewsEncoder(args)
-        self.user_encoder = UserEncoder(args)
+        self.news_encoder = NewsEncoder(
+            config.pretrained_model,
+            config.num_attention_heads,
+            config.additive_attn_hidden_dim,
+        )
+        self.user_encoder = UserEncoder(config.hidden_size, config.num_attention_heads)
```

Comment:
We made several similar changes along these lines, where a single object bundling many values together is expanded out to list the elements/values that are actually being used. That makes it easier to see how the values flow through the code and decouples the implementation of the function/method/class from the implementation of e.g. the config.